webob/request.py

```diff
@@ -892,37 +890,23 @@ def copy_get(self):
     # this way we can have seekable input without testing the .seek() method
     is_body_seekable = environ_getter('webob.is_body_seekable', False)
 
-    #is_body_readable = environ_getter('webob.is_body_readable', False)
-
-    def _is_body_readable__get(self):
+    @property
+    def is_body_readable(self):
         """
-            webob.is_body_readable is a flag that tells us
-            that we can read the input stream even though
-            CONTENT_LENGTH is missing. This allows FakeCGIBody
-            to work and can be used by servers to support
-            chunked encoding in requests.
-            For background see https://bitbucket.org/ianb/webob/issue/6
+            webob.is_body_readable is a flag that tells us that we can read the
+            input stream even though CONTENT_LENGTH is missing.
```

Comment:
You are falling back to `webob.is_body_readable` if `content_length == 0`, is that really what you want?